src/wasm-validator.h

```diff
@@ -27,14 +27,15 @@
 namespace wasm {
 
 struct WasmValidator : public PostWalker<WasmValidator, Visitor<WasmValidator>> {
-  bool valid;
+  bool valid = true;
+  bool validateWeb = false;
```

Comment:
Done, but renamed the original `validateWebConstraints` because they can't match.